galaxy/wrapper/test-data/plotPCA_result2.tabular

```diff
@@ -0,0 +1,3 @@
+Component	bowtie2-test1.bam	bowtie2-test1.bam	Eigenvalue
+1	-0.707106781187	-0.707106781187	2.0
+2	-0.707106781187	0.707106781187	0.0
```

Comment:
I personally find this convenient for using in R, since it makes a convenient data frame (obviously you'd want differently named files) :)